extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -69,34 +69,30 @@ class AmpAccordion extends AMP.BaseElement {
           'Each section must have exactly two children. ' +
           'See https://github.com/ampproject/amphtml/blob/master/extensions/' +
           'amp-accordion/amp-accordion.md. Found in: %s', this.element);
-      this.mutateElement(() => {
```

Comment:
see comment below, this can still happen but limited to session storage part now. `markMyChildrenAsDirty` is a fine solution but until then we still need a `mutateElement` around the part that adds `expanded` attribute (given that causes FOUC anyway, it should not be a big deal) or maybe `scheduleLayout()` if that's better than `mutateElement`?